examples/metrics/grafana-dashboards/strimzi-kafka-connect.json

```diff
@@ -754,5 +1046,5 @@
 
   "title": "Strimzi Kafka Connect",
   "uid": "rpITwc0mz",
-  "version": 6
-}
+  "version": 7
```

Comment:
I guess it's the Grafana dashboard version. Because we are not versioning dashboard yet, I would not change it for now but it's a good point for the future.

Reply:
Does that mean you want me to change this back to version 6 even if it's ignored?

Reply:
Yes please.